packages/services/src/boost.ts

```diff
@@ -75,6 +90,80 @@ export class BoostService {
     });
   }
 
+  claimStakedBoost(userId: string, wallet: string, competitionId: string) {
```

Comment:
new logic for activating boost. We loop through all stakes of a wallet and call `awardForStake`.